presto-main/src/main/java/com/facebook/presto/memory/MemoryResource.java

```diff
@@ -48,4 +50,12 @@ public MemoryInfo getMemoryInfo(MemoryPoolAssignmentsRequest request)
         taskManager.updateMemoryPoolAssignments(request);
         return memoryManager.getInfo();
     }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public MemoryInfo getMemoryInfo()
+    {
+        taskManager.updateMemoryPoolAssignments(new MemoryPoolAssignmentsRequest("", 0, new ArrayList<>()));
```

Comment:
What does this call do? I read through the code and this seems to do nothing since the request is empty.